source/EngineInterface/DescriptionEditService.cpp

```diff
@@ -97,7 +97,7 @@ CollectionDescription DescriptionEditService::createUnconnectedCircle(CreateUnco
             if (dxMod * dxMod + dy * dy > radiusRounded * radiusRounded + NEAR_ZERO) {
                 continue;
             }
-            result.addCell(CellDescription()
+            result.cells().push_back(CellDescription()
```

Comment:
`cells().push_back(CellDescription()...` won't work since cells() returns a reference to CollectionDescription.
Instead place it with `result._cells.emplace_back(CellDescription()...` as suggested. Same for other places.